tests/test_api_emails.py

```
@@ -0,0 +1,545 @@
+    data = response.json()
+    assert "service" in data
+    assert "version" in data
+    assert "status" in data
+
+
+@pytest.mark.asyncio
+async def test_health_check_endpoint():
+    """Test health check endpoint"""
+
+    class FakeSession:
+        async def __aenter__(self):
```

Ruff (0.14.10)

Missing return type annotation for special method `__aenter__`

(ANN204)